### PR TITLE
Add Facilities API release note for mobile and active_status

### PR DESCRIPTION
**Pull request by jeff2d2**
This adds a new release note to the dev portal for a change to the Facilities API response to include flags for `mobile` and `active_status`.

Addresses https://github.com/department-of-veterans-affairs/vets-contrib/issues/3341

### CODE REVIEW

**Comment by jeff2d2** [comment]
Preview: 
image

**Comment by nfasulo** [comment]
Not a required addition, but I wonder if we should add something about how you shouldn't trust the address listed for a mobile facility? The documentation says: "To get the exact current location, end consumers should contact the mobile facility directly."

File: src/content/apiDocs/facilities/facilitiesReleaseNotes.mdx [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## September 30, 2019
+You can now see if Health facilities are mobile or temporarily closed (for power outages, weather, etc.)
+- Added `mobile` and `active_status` flags to the API response for Health facilities (other facility types will return `null` for these fields). Options for `mobile` are `true` or `false`; options for `active_status` are `A` for Active or `T` for Temporarily closed. [View code change(s)](https://github.com/department-of-veterans-affairs/vets-api/pull/3339/commits)
+  - If a facility is mobile, the address is subject to frequently change. To get the exact current location, consumers should contact the mobile facility directly.
```
Comment (nfasulo) [inline src/content/apiDocs/facilities/facilitiesReleaseNotes.mdx]:
```suggestion
  - If a facility is mobile, the address is subject to frequent changes. To get the exact current location, consumers should contact the mobile facility directly.
```

**Comment by davem-va** [comment]
Thanks @jeff2d2 

Temporarily closed may not be correct.  VHA refers to this as temporarily deactivated.  This may make more sense.  My understanding is that the facility is not accepting appointments while in this "T" state but there may be medical staff reporting for duty so it's not really "closed."

Also, the #'s listed for mobile facilities are sometimes those for the mobile facility itself and other times for the parent facility.  To avoid confusion, should we say to call the number listed rather than "call the mobile facility directly?" It may not be possible to speak with the mobile facility directly in all cases.

**Comment by jeff2d2** [comment]
@davem-va Thank you. The language has been updated based on your feedback. I removed the part about *why* a facility might show as temporarily deactivated as it can be many reasons, and I didn't want to mention "weather or power outages" as that implies someone might not be there when that may not be the case.

**Pull request merged**